src/functions-and-arrays.js

```diff
@@ -52,14 +64,30 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
 
 
+const uniquifyArray = (arr) => {
+  let newArr = [];
+  arr.length === 0 ? newArr = null : arr.forEach(x => newArr.indexOf(x) === -1 ? newArr.push(x) : x)
+  return newArr;
+}
+
```

Comment:
- Intenta utilizar siempre nombres descriptivos e intenta evitar nombres tales como `x`, `y`... pueden ahorrar espacio pero no vale la pena en comparación a la legilibidad que se pierde:
- Te dejo con dos posibilidades:

```suggestion
const uniquifyArray = (arr) => {
  let newArr = [];
  arr.length === 0 ? newArr = null : arr.forEach(elm => newArr.indexOf(x) === -1 ? newArr.push(elm) : elm)
  return newArr;
}
```

```suggestion
const uniquifyArray = (value) => {
  let newArr = [];
  arr.length === 0 ? newArr = null : arr.forEach(value=> newArr.indexOf(x) === -1 ? newArr.push(value) : value)
  return newArr;
}
```